sql/2021/caching/resource_age_party_and_type_wise_groups.sql

```diff
@@ -0,0 +1,52 @@
+#standardSQL
+# Age of resources party, type wise in groups.
+CREATE TEMPORARY FUNCTION toTimestamp(date_string STRING)
+RETURNS INT64 LANGUAGE js AS '''
+  try {
+    var timestamp = Math.round(new Date(date_string).getTime() / 1000);
+    return isNaN(timestamp) || timestamp < 0 ? null : timestamp;
+  } catch (e) {
+    return null;
+  }
+''';
+
+SELECT
+  client,
+  party,
+  resource_type,
+  COUNT(0) AS requests_total,
```

Comment:
As `WHERE` clause below it won't be total. But maybe you're OK with being total with `resp_last_modified`?